.pylintrc

```diff
@@ -74,9 +74,6 @@ enable=import-error,
        global-at-module-level,
        bad-open-mode,
        redundant-unittest-assert,
-       # Has common issues with our style due to
-       # https://github.com/PyCQA/pylint/issues/210
-       unused-variable,
```

Comment:
The new version of pylint seems to be better at finding unused variables and I didn't want to fix them all in this branch.